app/install/centos.md

```diff
@@ -11,15 +11,15 @@ breadcrumbs:
 
 Start by downloading the corresponding package for your configuration:
 
-- [CentOS 6]({{ site.links.download }}/{{site.data.kong_latest.version}}/kong-{{site.data.kong_latest.version}}.el6.noarch.rpm)
-- [CentOS 7]({{ site.links.download }}/{{site.data.kong_latest.version}}/kong-{{site.data.kong_latest.version}}.el7.noarch.rpm)
+- [CentOS 6]({{ site.links.download }}/kong-community-edition-rpm/download_file?file_path=dists/kong-community-edition-{{site.data.kong_latest.version}}.el6.noarch.rpm)
+- [CentOS 7]({{ site.links.download }}/kong-community-edition-rpm/download_file?file_path=dists/kong-community-edition-{{site.data.kong_latest.version}}.el7.noarch.rpm)
 
 ### YUM Repositories
 
-You can also install Kong by using the following YUM repositories and following the Bintray instructions:
+You can also install Kong via YUM; follow the instructions on the "Set Me Up"
+section on the page below.
 
-- [CentOS 6 YUM](https://bintray.com/mashape/kong-rpm-el6-{{site.data.kong_latest.release}})
-- [CentOS 7 YUM](https://bintray.com/mashape/kong-rpm-el7-{{site.data.kong_latest.release}})
+- [RPM Repository](https://bintray.com/kong/kong-community-edition-rpm)
```

Comment:
* Concerning the YUM and APT repositories, a better "call to action" would be to directly inline the repositories sources and instructions. 
* Granted, the scope of this PR is not to provide inline repositories setup instructions, only to update the links - just a thought!
* Additionally, the instructions specified by bintray don't include the obligatory `apt-get update` step, or is that step not needed? Or are we counting on the below section: `If you are downloading the package, execute: sudo apt-get-update`? I feel like the "Install Kong" section could be split between manual download of the package, and download via the YUM/APT repositories. Not in this PR's scope either.